src/Types/ApiFormData.ts

```diff
@@ -37,18 +37,13 @@ export type ApiHouseholdMemberReadOnly = {
 };
 
 export type ApiHouseholdMember = {
-  relationship: string | null;
   age: number | null;
+  relationship: string | null;
   student: boolean | null;
-  student_full_time?: boolean | null;
   pregnant: boolean | null;
-  unemployed?: boolean | null;
-  worked_in_last_18_mos?: boolean | null;
   visually_impaired: boolean | null;
   disabled: boolean | null;
-  veteran?: boolean | null;
-  medicaid?: boolean | null;
-  disability_medicaid?: boolean | null;
```

Comment:
Can we wait until I change the API to make these changes to the API types so that I can remove them all at one time.

Reply:
Yes. I've added them back in to this PR. It looks like @sydney-devine created a ticket for us to do this: https://github.com/Gary-Community-Ventures/benefits-api/issues/158